more/free-programming-cheatsheets.md

```diff
@@ -289,7 +291,9 @@
 
 ### Machine Learning
 
+* [Cheat Sheet of Machine Learning and Python (and Math) Cheat Sheets](https://medium.com/machine-learning-in-practice/cheat-sheet-of-machine-learning-and-python-and-math-cheat-sheets-a4afe4e791b6) - Medium (HTML)
 * [Machine Learning Cheat Sheet](https://www.datacamp.com/cheat-sheet/machine-learning-cheat-sheet) - DataCamp Team (HTML)
+* [Machine Learning Glossary](https://ml-cheatsheet.readthedocs.io/en/latest/index.html) - ml-cheatsheet (HTML)
```

Comment:
I think the author of this is Brendan Fortuner, per the GitHub link. 

I also noticed this link works without the `/index.html` which might be better.

Reply:
done